packages/playground/windows/playground-win32/Playground-Win32.cpp

```diff
@@ -27,6 +27,8 @@
 
 #pragma pop_macro("GetCurrentTime")
 
+#define BUFFER_LENGTH 6
```

Comment:
>BUFFER_LENGTH [](start = 8, length = 13)

In modern C++ we should avoid using macros for constants. It is better to use constexpr instead. 

I am guilty of adding a lot of such constants to ReactInstanceSettings - I promise to fix them soon. :)